### PR TITLE
Add data-testid attributes

### PR DESCRIPTION
**Pull request by rwood-moz**
Adding some `data-testid` attributes to some UI DOM elements that may be used by future E2E tests.

### CODE REVIEW

**Review by MelissaAutumn — APPROVED** [review]
Thank you!

Is there a general convention for these like "input-div-wrapper" and "input-div-wrapper-input" is something I'm seeing here? Or is it mostly just pick and choose?

File: frontend/src/views/SettingsView.vue [hunk 1]
```diff
@@ -94,6 +94,7 @@ onMounted(() => {
         "
         :class="{ '!bg-teal-500 !text-white': view === activeView }"
         @click="show(key)"
+        :data-testid=key
```
Comment (devmount) [inline frontend/src/views/SettingsView.vue:97]: Is there a reason to not using quotes here? I think we always use them even on single word attributes.

```suggestion
        :data-testid="key"
```

Comment (devmount) [reply frontend/src/views/SettingsView.vue:97]: Oh and the term `key` might be a bit too general as an identifier? But I'm really no expert here, if that is sufficient that's totally fine.

**Review by devmount — APPROVED** [review]
Looking good! Just wondering the same thing like Melissa, if we should follow a certain convention for the test id naming.

Thank you for taking care of this!!

**Comment by rwood-moz** [comment]
Appreciate your feedback @MelissaAutumn and @devmount! PR is updated and ready for re-review when you have some time, thanks!

**Review by devmount — APPROVED** [review]
Awesome, thanks for the changes! Can be merged imo

**Pull request merged**